src/_tutorials/dart-vm/httpserver.md

```diff
@@ -629,13 +623,14 @@ for example, the URI has no query string
 * has no state and does not change the state of the server
 * has no length limits
 
-<strong>Bonus code:</strong>
-If you would like to see some client code that
-makes GET requests,
-check out the code for
-[number_guesser.dart.]({{gh-path}}/bin/number_guesser.dart)
-It's a standalone client for the number thinker server
-that makes periodic guesses until it guesses correctly.
+<aside class="alert alert-info" markdown="1">
+  If you would like to see some client code that
```

Comment:
We can make this shorter (not your text, I know) and add a title that makes the transition clearer. (I don't know, offhand, why this section about POST requests is pointing you to a GET example.) Something like this (but better, this is a WAG):

**Curious about how GET and POST differ?** To see client code that makes GET requests instead of POST requests, look at .... Compare it to....

Reply:
I've dropped the \<aside> (falling back to regular paragraphs), replacing it with:

> The client in this example makes REST-compliant POST requests.
>
> To see client code that makes REST-compliant GET requests
look at [number_guesser.dart.]({{gh-path}}/bin/number_guesser.dart)
It's a standalone client for the number thinker server
that makes periodic guesses until it guesses correctly.

WDYT?